yetanotherco/aligned_layer
# feat: add to sdk an option to wait for the verification of the batcher in ethereum make it the default

# Description

- This PR adds the option for the client to wait for verification in ethereum.

# To Test

- Start anvil with block time, aggregator, operator and batcher.
- Run `make install_aligned_compiling` and `which aligned` to make sure you are using the `.cargo/bin` version.
- Go to `/batcher/aligned-sdk` and run `cargo test`. Make sure to keep sending more proofs with `make batcher_send_groth16_bn254_task` or another make target so the batches fill with more than one proof.
- Send more proofs with the normal flow and try the different bursts.
